larpandoracontent/LArObjects/LArMCParticle.h

```diff
@@ -168,15 +253,55 @@ inline pandora::StatusCode LArMCParticleFactory::Write(const Object *const pObje
     if (!pLArMCParticle)
         return pandora::STATUS_CODE_INVALID_PARAMETER;
 
+    const std::vector<pandora::CartesianVector> &mcStepPositions = pLArMCParticle->GetMCStepPositions();
+    int nMCStepPositions = mcStepPositions.size();
+    std::cout << "LArMCParticleFactory::Write - nMCStepPositions = " << nMCStepPositions << std::endl;
```

Comment:
Can drop the debug calls, if required. I'm actually not sure what the precedent is around this for writing PNDR/XML files.